### DISCUSSION TITLE
hive-connector 无法在hive上执行聚合计算

### DISCUSSION THREAD

**Discussion by chariswang188996** [apache/iotdb#3735] [Q&A]
Hi,我有一个hadoop集群，hadoop的版本是2.7.7（和官网给出的2.7.3的版本有出入），hive版本是2.7.3.通过源码编译好的hive-connector通过官网介绍的方式，加载到hive里，普通的sql执行可以得出结果。
![图片1](https://user-images.githubusercontent.com/12324219/129432485-2eb299e3-0a9d-4dec-98c6-0256788eff54.png)

但是执行聚合查询的时候返回0
![图片2](https://user-images.githubusercontent.com/12324219/129432486-bc4ce1b0-cf5b-4cc2-a063-cb6dd7ead590.png)

我想问一下hadoop的版本必须是2.7.3么？

**Comment by JackieTien97** [comment]
hive的版本官网给的经过测试可以运行的版本是2.3.6，你可以将hive版本换成这个试试。还可以把hive的具体报错日志给贴出来哈，应该hive有个配置项是可以指定错误日志的输出路径的

**Reply by chariswang188996** [reply]
这个是我们的人弄错了  我们的hive是2.3.6；您那边的hadoop的版本是多少？
<img width="694" alt="截屏2021-08-16 10 01 11" src="https://user-images.githubusercontent.com/12324219/129502054-9048e772-fd99-4717-bc4f-0e383fd37349.png">

我们查了一下 hive的log没有报错。我们使用的./hive --hiveconf hive.root.logger=DEBUG,console 这样的方式，执行sql没看到报错。
[hive-debug.log.zip](https://github.com/apache/iotdb/files/6989437/hive-debug.log.zip)

**Reply by qiaojialin** [reply]
解决了吗，什么错误导致的？

**Reply by chariswang188996** [reply]
> 解决了吗，什么错误导致的？

目前还没。 日志上看不到报错，聚合查询count(测点) 返回值是0，但是select 语句是能看到结果的
